tests/early-data.rs

```diff
@@ -113,13 +123,16 @@ async fn test_0rtt() -> io::Result<()> {
     let config = Arc::new(config);
     let addr = SocketAddr::from(([127, 0, 0, 1], server_port));
 
-    let (io, buf) = send(config.clone(), addr, b"hello").await?;
+    let (io, buf) = send(config.clone(), addr, b"hello", vectored).await?;
     assert!(!io.get_ref().1.is_early_data_accepted());
     assert_eq!("LATE:hello", String::from_utf8_lossy(&buf));
 
-    let (io, buf) = send(config, addr, b"world!").await?;
+    let (io, buf) = send(config, addr, b"world!", vectored).await?;
     assert!(io.get_ref().1.is_early_data_accepted());
     assert_eq!("EARLY:world!LATE:", String::from_utf8_lossy(&buf));
 
     Ok(())
 }
+
+// Include `utils` module
+include!("utils.rs");
```

Comment:
Why not just use `mod utils;`?

Reply:
I've done it the way the other files do it, which I think has to do mainly with `src/common/test_stream.rs` wanting to import it as well